src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/SubmitLicenceRequest.kt

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model
+
```

Comment:
Is this named clearly?  SubmitLicenceRequest that contains only the COMs details?

Reply:
I think this is just my understanding - I can see that the com's details are not now added on creation of the licence, but on submission of the licence to prison - and this request therefore makes more sense.